Utilities/Logging.cs

```diff
@@ -69,6 +70,7 @@ private static void log(ErrorLevel errorlevel, string data, string method, strin
                     // Failed; can't do anything about it as we're in the logging code anyway
                 }
             }
+            WriteTelemetry($"{shortPath}:{method}", data);
```

Comment:
Isn't this sending for *all* logging levels, not just the ones we want to send?

Reply:
Yes, this will provide context from the info and debug level logs written immediately prior to the error. I think this context will be useful.
Data is subject to the same redactions and filters as any other data sent to Rollbar and is not sent until an error is sent.

Reply:
If an exception is thrown before `shortPath` is initialized, we might get to line 73 with a null value, and I do not remember how `$` behaves with null values.

Reply:
It behaves as an empty string. Confirmed with the following test:
```cs
        [TestMethod]
        public void TestTemp()
        {
            string shortPath = null;
            Assert.AreEqual("", $"{shortPath}");
        }
```

Reply:
SG